utils/syntax_highlighting.tsx

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import hlJS from 'highlight.js/lib/highlight.js';
```

Comment:
probably we need to import this like `import hlJS from 'highlight.js';` to makes the tipes work correctly.